index.html

```diff
@@ -2999,7 +2997,7 @@ <h3>Zero-Knowledge Proofs</h3>
 <a>holder</a> and <a>subject</a> through the use of selective disclosure of the
 <a>verifiable credential</a> values. Some other cryptographic systems which rely
 upon zero-knowledge proofs to selectively disclose attributes can be found in the
-[[?LDP-REGISTRY]] as well.
+Verifiable Credential Specifications Directory [[?VC-SPECS]] as well.
```

Comment:
curious about the `?` - how can we normatively refer to this?

Reply:
AFAIK, we can't.

Reply:
`?` means it's an informative reference... we can't normatively reference an informative document, which VC-SPECS is. So, as @OR13 said, we can't normatively reference VC-SPECS; it is :ghost:W3C FORBIDDEN:ghost:.